### PR TITLE
[CDAP-3494] Feature/ui redesign explore

### PR DESCRIPTION
**Pull request by elfenheart**
cluster: newsinglenode2671-1000

newexplore

### CODE REVIEW

File: cdap-ui/app/directives/explore/explore.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+              isFirstTime = false;
+
+              vm.queries = queries;
+
+              // Polling for status
+              angular.forEach(vm.queries, function(q) {
+                q.isOpen = false;
+                if (q.status !== 'FINISHED') {
+
+                  // TODO: change to use myExploreApi once figure out how to manually stop poll with $resource
```
Comment (ajainarayanan) [inline cdap-ui/app/directives/explore/explore.js]: Is this still valid? We can stop a poll using ```$resource``` right?

File: cdap-ui/app/directives/explore/explore.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+        vm.getQueries();
+
+        vm.preview = function (query) {
+          $bootstrapModal.open({
+            templateUrl: 'explore/preview-modal.html',
+            size: 'lg',
+            resolve: {
+              query: function () { return query; }
+            },
+            controller: ['$scope', 'myExploreApi', function ($scope, myExploreApi) {
```
Comment (ajainarayanan) [inline cdap-ui/app/directives/explore/explore.js]: We resolve ```query``` but not using it in the controller?

File: cdap-ui/app/directives/explore/explore.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+          myExploreApi.postQuery(params, { query: vm.query }, vm.getQueries);
+          vm.currentPage = 1;
+        };
+
+        vm.getQueries = function() {
+
+          myExploreApi.getQueries(params)
+            .$promise
+            .then(function (queries) {
+              if (!isFirstTime) {
```
Comment (ajainarayanan) [inline cdap-ui/app/directives/explore/explore.js]: Not sure if we require ```isFirstTime``` and the ```if``` block. Just the ```$timeout``` block should be fine right?

Comment (elfenheart) [inline cdap-ui/app/directives/explore/explore.js]: The if block will prevent all the queries being green the first time user loads the page.

**Comment by ajainarayanan** [comment]
:+1: LGTM

**Pull request merged**